rinja_actix/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rinja_actix"
-version = "0.3.0"
+version = "0.3.1"
```

Comment:
A reason why you're updating versions? Shouldn't it be done in a separate PR?

Reply:
Because I added a new feature, but it could be done in another PR, too.

Reply:
It's not the first one. Please do it in another PR then. Like that we make publish it too and also ensure we don't have any other features we want to be merged before making this release.